lib/fluent/plugin/out_copy_ex.rb

```diff
@@ -1,71 +1,43 @@
-require 'fluent/output'
+require 'fluent/plugin/multi_output'
+require 'fluent/config/error'
+require 'fluent/event'
 
-module Fluent
+module Fluent::Plugin
```

Comment:
Do we need to change the namespace here? If this is not necessary, could you keep as is?

Reply:
I don't know. I copied the out_copy.rb file from fluentd: https://github.com/fluent/fluentd/blob/e2e0f64e006605cb0d47e25572913af857cda082/lib/fluent/plugin/out_copy.rb#L21